app/models/game.rb

```diff
@@ -1,3 +1,34 @@
 class Game < ActiveRecord::Base
   scope :available, -> { where('black_player IS NULL OR white_player IS NULL') }
+  has_many :users
+  has_many :pieces
+
+  def populate_board!
```

Comment:
Just refactored it from around 30 to 15 lines and set the rubocop method length back to 20. It's still just 1 method, but had to pass it 2 parameters - vertical_position & color.